packages/vega/src/VegaPlot.js

```diff
@@ -24,9 +26,54 @@ export function VegaPlot(props) {
   const {
     spec: partialSpec,
     data,
+    setName,
     signalListeners,
   } = props;
 
+  // eslint-disable-next-line no-unused-vars
+  const classes = useStyles();
+
+  const generateCSS = () => `
+    .tooltip-container {
+      display: flex;
+      justify-content: space-between;
+    }
+    .tooltip-item-name {
+      margin-right: 10px;
+      font-weight: bold;
+    }
+    .tooltip-item-value {
+      margin-left: auto;
+    }
+  `;
+
+  const tooltipConfig = {
+    theme: 'custom',
+    offsetX: 10,
+    offsetY: 10,
+    formatTooltip: e => `
+      <style>${generateCSS()}</style>
+      <div class="tooltip-container">
+        <span class="tooltip-item-name">${setName} Set </span><div class="tooltip-item-value">${e.name}</div>
+      </div>
+      <div class="tooltip-container">
+        <span class="tooltip-item-name">${setName} Set Size</span><div class="tooltip-item-value">${e.size}</div>
```

Comment:
This logic should be lifted up into the "consumer components" of `VegaPlot` (`CellSetSizesPlot`, `CellSetExpressionPlot`, etc), to keep `VegaPlot` agnostic to the Vega/Vega-Lite spec that it is used to render.

Reply:
Very good point! This is done now.